src/context/yaml/handlers/brandingTemplates.js

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs-extra';
```

Comment:
Are you actually interested in creating a separate set of handlers for branding templates? It would seem to me that branding templates fall squarely within the realm of branding. And judging by the [deletion of the `templates` property](https://github.com/auth0/auth0-deploy-cli/pull/438/files#diff-1b948ca54d6b7880eb055d92d1c07525248d308e455ccae5660c01e4473065e3R33), it looks like there is a shared concern between these two handlers.

We typically try to conceptually tie the resource types in our config-management repos with the [Management API entities](https://auth0.com/docs/api/management/v2). So in this case, I'd really love to see it all bundled underneath a generalized branding umbrella if you can.

Reply:
100%. Actually, I didn't realize that the templates were a "sub-entity" of branding until I'd already started, and then didn't ever re-evaluate the file structure.

I can easily just add the new template logic to the existing branding handler, and update the test filenames to be consistent as well. I don't anticipate any functional changes, so if it's OK with you I will just squash the change into the existing commit.

Reply:
> Actually, I didn't realize that the templates were a "sub-entity" of branding until I'd already started

Agreed that the hierarchy isn't obvious, but it rids us of needing to make that decision ourselves.

> so if it's OK with you I will just squash the change into the existing commit

Yep, all good with me.